docs/UserGuide.md

```diff
@@ -50,158 +70,247 @@ Some example commands you can try:
-</div>
+**Example & Output:** `add n/All of us are dead s/plan-to-watch t/Horror`
+
+[INSERT IMAGE AFTER UI IS DONE COMPLETELY]
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
 
 ### Deleting a show: `delete`
 
-Deletes the specified show from Trackermon.
+**Description:** Wanting to remove an unwanted show? Delete the show at the specified index shown in Trackermon's show list!
+
+**Format:** `delete INDEX`
+
+**Example & Output:** `delete 2`
 
-Format: `delete <INDEX>`
-* Deletes the show at the specified `<INDEX>`.
-* The index refers to the index number shown in the displayed show list. (not overall)
-* The index **must be a positive integer** 1,2,3,..
+[INSERT IMAGE AFTER UI IS DONE COMPLETELY]
 
-Examples:
-* `list` followed by `delete 2` removes 2nd show in Trackermon.
-* `find ghibli` followed by `delete 1` removes 1st show in results of `find` command.
+<div markdown="block" class="alert alert-danger"> **:exclamation: Caution:** 
+The **index** parameter provided should be a [non-zero unsigned integer](https://en.wikipedia.org/wiki/Integer_(computer_science)) within the allowed range of Java’s [`int`](#glossary) data type. On top of that, the index should be within the bounds of the show list.<br>
+  e.g. If there are 5 shows saved in the show list, then the valid index ranges from 1 to 5.
+
+</div>
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
 ---
-### Editing a show: `Edit`
 
-Edit the specified show from Trackermon.
+### Editing a show: `edit`
+
+**Description:** Wanting to modify a show? Edit a show at the specified index shown in Trackermon's show list!
+
+**Format:** `edit INDEX [n/NAME] [s/STATUS] [c/COMMENT] [t/TAG]…​`
+
+**Example & Output:** `edit 2 n/Sailor Moo`
 
-Format: `edit <INDEX> [n/<NAME>] [s/<STATUS>] [t/<TAG>]…​`
-* Edit the show at the specified `<INDEX>`.
-* The index refers to the index number shown in the displayed show list. (not overall)
-* The index **must be a positive integer** 1,2,3,..
-* At least one field to edit must be provided
-* Editing to an existing name is not allowed
+[INSERT IMAGE AFTER UI IS DONE COMPLETELY]
 
-Examples:
-* `list` followed by `edit 2 n/Sailor Moo` edit 2nd show's name in Trackermon to Sailor Moo.
-* `find ghibli` followed by `edit 1 n/Cowman s/completed t/awesome` edits 1st show in results of `find` command.
+<div markdown="block" class="alert alert-danger">
+
+**:exclamation: Caution:**<br>
+* **At least one field** to edit must be provided.
+* Editing to an existing name is **not allowed**.
+* The **index** parameter provided should be a [non-zero unsigned integer](https://en.wikipedia.org/wiki/Integer_(computer_science)) within the allowed range of Java’s [`int`](#glossary) data type. On top of that, the index should be within the bounds of the show list.<br>
+  e.g. If there are 5 shows saved in the show list, then the valid index ranges from 1 to 5.
+
```

Comment:
Was wondering if we should inform the user that editing the tags would cause all of the previous tags to be deleted. Would like to hear your opinion about it!

Reply:
Good point. I will add that in!